templates/boot/config.js

```diff
@@ -14,7 +14,7 @@ var publicDir = path.join(assetsDir, 'public');
 var templatesDir = path.join(assetsDir, 'emails');
 var viewsDir = path.join(appDir, 'views');
 
-var maxAge = 24 * 60 * 60 * 1000;
+var maxAge = 365 * 24 * 60 * 60;  // One year in seconds
```

Comment:
Are we sure that for `session` it's in seconds? I had the feeling [it should be in milliseconds](https://github.com/expressjs/session#cookiemaxage).